<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HackProcessWindow</class>
 <widget class="QDialog" name="HackProcessWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>372</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hack Process</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="processNameLabel">
     <property name="text">
      <string>Process name:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="processNameLineEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Integrity level:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="integrityLevelComboBox">
     <property name="editable">
      <bool>false</bool>
     </property>
     <item>
      <property name="text">
       <string>Untrusted</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Low</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Medium</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>High</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>System</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>List of used libraries:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="librariesTable">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
